config/initializers/assets.rb

```diff
@@ -0,0 +1,2 @@
+Rails.application.config.assets.precompile += %w( govuk_admin_template/favicon-development.png )
+Rails.application.config.assets.precompile += %w( govuk_admin_template/favicon.png )
```

Comment:
These lines won't be necessary when this PR will be merged and a new gem version will be released:
https://github.com/alphagov/govuk_admin_template/pull/119